os/include/tinyara/binfmt/binfmt.h

```diff
@@ -385,7 +385,7 @@ int binfmt_exit(FAR struct binary_s *bin);
  *
  ****************************************************************************/
 #ifdef CONFIG_BINARY_MANAGER
```

Comment:
This ifdef can be completely removed and added into Makefile. Coz, all files inside the binary_manager should be compiled only when this flag is enabled.

Reply:
@btheosam  ELF loader can work without binary manager, so hence this is required.